src/services/mock/generate

```diff
@@ -20,7 +20,7 @@ if ! sed --version 2>/dev/null | grep -q 'GNU' 2>/dev/null; then
 fi
 
 cd "$(dirname "$0")" || exit 1
-CHECK_FILE='checkpoint.js'
+CHECK_FILE='json/OnWorkflowTreeDeltasData.json'
```

Comment:
Didn't test, but now it should be simpler. Just dump whatever is returned from GraphQL into a JSON file, and let `json-server` serve it. We could even have more requests, use a URL with params for tests like `/graphql?throwException` or `/graphql?returnInvaildDelta`, to return a different JSON.